lightbulb/command_handler.py

```diff
@@ -169,6 +175,27 @@ def __init__(
 
         self._help_impl = help_class(self)
 
+        self.extensions_path = extensions_path
+        if self.extensions_path:
+            self.full_extensions_path = pathlib.Path.cwd() / self.extensions_path.replace(".", os.sep)
+
+            self.subscribe(hikari.StartedEvent, self.start_extension_watch)
+
+    async def start_extension_watch(self, _) -> None:
+        async for changes in awatch(self.full_extensions_path):
```

Comment:
There is no need to store `full_extensions_path` as you can just generate it here at the begging from `extensions_path`

PS: After checking the documentation, you can specify a relative path, so there is no need to generate a fully qualified path.

Reply:
Ah alright